packages/hdwallet-native/src/cosmos.ts

```diff
@@ -0,0 +1,108 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+import { addressNListToBIP32, CosmosSignTx, CosmosSignedTx } from "@shapeshiftoss/hdwallet-core";
+
+let txBuilder = require("@bithighlander/cosmos-tx-builder");
+import HDKey from "hdkey";
+const bip39 = require(`bip39`);
+const ripemd160 = require("crypto-js/ripemd160");
+const CryptoJS = require("crypto-js");
+const sha256 = require("crypto-js/sha256");
+const bech32 = require(`bech32`);
+
+function bech32ify(address, prefix) {
+  const words = bech32.toWords(address);
+  return bech32.encode(prefix, words);
+}
+
+function createCosmosAddress(publicKey) {
```

Comment:
usually just put the helper functions within the class using them as they are only used in one